app/static/js/createEvents.js

```diff
@@ -173,48 +184,91 @@ $('#multipleOfferingSave').on('click', function() {
     }
   }
 
+  // Add past date validation
+  datePickerInputs.each(function(index, element) {
+    if (!allowPastStart && isDateInPast($(element).val())) {
+      $(element).addClass('border-red');
+      hasPastDates = true;
+    } else {
+      $(element).removeClass('border-red');
+    }
+  }); 
+
   if (isEmpty){
     let emptyFieldMessage = "Event name or date field is empty";
     displayNotification(emptyFieldMessage);
   }
-  else if(!hasValidTimes){
+  else if (!hasValidTimes) {
     let invalidTimeMessage = "Event end time must be after start time";
     displayNotification(invalidTimeMessage);
   }
-  else if (hasDuplicateListings){
+  else if (hasDuplicateListings) {
     let eventConflictMessage = "Event listings cannot have the same event name, date, and start time";
     displayNotification(eventConflictMessage);
   }
+  else if (hasPastDates) {
+    showModalFlashMessage ("Some events have dates in the past. Please correct them or enable 'Allow start date to be in the past'.", "danger");
+  }
   else {
-    saveOfferingsFromModal();
+    let offerings = [];
+    eventOfferings.each(function(index, element) {
+      offerings.push({
+        eventName: $(element).find('.multipleOfferingNameField').val(),
+        eventDate: $(element).find('.multipleOfferingDatePicker').val(),
+        startTime: $(element).find('.multipleOfferingStartTime').val(),
+        endTime: $(element).find('.multipleOfferingEndTime').val()
+      });
+    });
+
+    let offeringsJson = JSON.stringify(offerings);
+    $("#multipleOfferingData").val(offeringsJson);
+
     updateOfferingsTable();
     pendingmultipleEvents = [];
     $("#checkIsMultipleOffering").prop('checked', true);
-    // Remove the modal and overlay from the DOM
     $('#modalMultipleOffering').modal('hide');
-    $('.invalidFeedback').css('display', 'none');
-    $('#textNotifierPadding').removeClass('pt-5');
     msgFlash("Multiple time offering events saved!", "success");
   }
 });
-
+function showModalFlashMessage(message, type) {
+  $('#textNotifier').removeClass().addClass('alert alert-' + type).text(message).show();
+  $('#textNotifierPadding').addClass('pt-5');
+  setTimeout(function() {
+    $('#textNotifier').hide();
+    $('#textNotifierPadding').removeClass('pt-5');
+  }, 5000);
+}
 
 // Save the offerings from the modal to the hidden input field
 function saveOfferingsFromModal() {
```

Comment:
I'm not sure I understand this

Reply:
Since you deleted the call to this function, there is nowhere else in the code base that uses this function. When I checked out the branch, the function was even grayed out. Can the function be removed?